.github/workflows/ci.yml

```diff
@@ -33,8 +33,8 @@ jobs:
 
           flake8 .
 
-      - name: Run nonstandardcode.py
+      - name: Run housing_price.py
 
         run: |
 
-          python nonstandardcode.py
+          python src/house_package_saachi/housing_price.py

```

Comment:
You need not keep modifying the old _workflow files_ to accommodate for the new changes in the _repo_.
So every time you add a new _workflow_ just suppress the previous _workflow_'s trigger like the following..
```
on:
  pull_request:
    branches: dummy_branch
```

Reply:
Modified ci.yml to its supress workflow